config/routes.rb

```diff
@@ -1,11 +1,9 @@
 Rails.application.routes.draw do
+  resources :recipes, only: [:index, :show]
+
   get 'ingredients/index'
 
   root 'ingredients#index'
 
-  get 'recipes/show'
```

Comment:
The problem is not in routing. It's not a crud action. You need to fix either controller /view or add random specific routes. I will show you in the office.